### DISCUSSION TITLE
Book Structure

### DISCUSSION THREAD

**Discussion by sfarrens** [CosmoStat/Shear-and-PSF-Reading-Group#23] [Ideas]
Following today's discussion here is a summary of what was suggested:

1. Add a subsection on detection to the book introduction giving a general overview of how the information used for shape measurement is extracted from the observed image pixels.
1. Add a chapter called PSF with sections on:
   - A definition of what the PSF is and what sources contribute to the observed aberrations
   - PSF estimation with subsections on:
     - Parametric methods
     - Non-parametric methods
1. Add sections to the Shape Measurement chapter on:
   - Model-fitting methods
   - Bayesian methods
   - Machine learning methods
1. Add subsections to the Shear Bias section on:
   - Defining what shear bias is and the different types
   - Selection bias
   - Detection bias
1. Add subsections to the Shear Calibration section on:
   - Simulations
   - Metacalibration
   - Metadetection

**Comment by aboucaud** [comment]
I will add to that the subject on **blending**.
We suggested and agreed that blending should not make up one section but rather appear within the othere sections where it is relevant (measurement, bias, calibration, ...)
